koa-crud/src/pagination/index.ts

```diff
@@ -0,0 +1,41 @@
+import R from 'ramda';
+
+const paginate = (data, first, after) => {
```

Comment:
Limit the argument to  two only. if it is more than two then make the argument an `object` instead

Reply:
copy sir